pkg/trace/pb/stats.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
```

Comment:
Shouldn't the proto definition be removed to as per https://github.com/DataDog/datadog-agent/pull/6687#discussion_r537476515